lib/permutations.rb

```diff
@@ -1,4 +1,21 @@
+#permutations?("hello", "ehllo") => true
+#permutations?("pasta", "atsap") => true
+#permutations?("Pizza", "Pasta") => false
+#permutations?("", "") => true
+#Write a method which will take two strings as arguments and returns true if one string is a permutation of the other.
 
 def permutations?(string1, string2)
-  raise NotImplementedError, "permutations? not implemented"
-end
\ No newline at end of file
+
+  if string1 != string2
+    return false
+  end
+  
+  string1 = string1.chars.sort.join
+  string2 = string2.chars.sort.join
```

Comment:
This works, but it's inefficient because you're doing a sort and not using a hash.